src/data_parser.py

```diff
@@ -1,20 +1,47 @@
 import data_fetcher
+import json
```

Comment:
Please group and order imports by standard libs, third party libs, and local libs, per [PEP 8 guidelines](https://peps.python.org/pep-0008/#imports)

Reply:
Noted, thank you for the link to the documentation.